pkg/testing/tools/estools/elasticsearch.go

```diff
@@ -375,7 +375,43 @@ func CheckForErrorsInLogs(client elastictransport.Interface, namespace string, e
 // CheckForErrorsInLogsWithContext checks to see if any error-level lines exist
 // excludeStrings can be used to remove any particular error strings from logs
 func CheckForErrorsInLogsWithContext(ctx context.Context, client elastictransport.Interface, namespace string, excludeStrings []string) (Documents, error) {
+	excludeStatements := []map[string]interface{}{}
+
+	// Some if not all of those processors are likely to always log some error
+	// just because they cannot fetch the information, like if docker is not running
+	// or the Beat is not running on Kubernetes, so we already exclude all errors from them.
+	processors := []string{
+		"add_host_metadata",
+		"add_cloud_metadata",
+		"add_docker_metadata",
+		"add_kubernetes_metadata",
+	}
```

Comment:
Hmm I'm not sure I agree with unconditionally ignoring every error from these processors.

This would miss legitimate regressions, for example if the add_host_metadata processor were to start reporting errors when we added the FQDN feature.

I would rather us exclude specific errors.

Reply:
It looks like we are filtering the specific errors on main now

https://github.com/elastic/elastic-agent/blob/a0e49afa6788917e713dd6127799ae5199bcdbaa/testing/integration/logs_ingestion_test.go#L140-L144

Is this change still necessary?

Reply:
I wasn't sure if that was the best approach. I believe it is a bit hard to extensively exclude all possible expected errors from those loggers (or any other).

I agree it might be too broad. I'm fine with closing this PR if it has already been fixed in a different way.